frontend/src/bundles/auth/store/slice.ts

```diff
@@ -25,8 +25,9 @@ const { reducer, actions, name } = createSlice({
         builder.addCase(signUp.pending, (state) => {
             state.dataStatus = DataStatus.PENDING;
         });
-        builder.addCase(signUp.fulfilled, (state) => {
+        builder.addCase(signUp.fulfilled, (state, action) => {
             state.dataStatus = DataStatus.FULFILLED;
+            state.user = action.payload;
```

Comment:
```suggestion
            state.user = action.payload.user;
```

Reply:
As i understand token is saved in localStorage, not in store, as action payload include user and token

Reply:
yes, do like this

```
const signIn = createAsyncThunk<
    AuthResponseDto,
    UserAuthRequestDto,
    AsyncThunkConfig
>(`${sliceName}/sign-in`, async (loginPayload, { extra }) => {
    const { authApi } = extra;
    const response = await authApi.signIn(loginPayload);
    if (response.token) {
        await storage.set(StorageKey.TOKEN, response.token);
    }
    return response;
});```

Reply:
Yes, it was already done, didn't you see it ?